lldb/test/API/functionalities/postmortem/elf-core/TestLinuxCore.py

```diff
@@ -728,7 +746,6 @@ def test_riscv64_regs(self):
             "fa2",
             "fa3",
             "fa4",
-            "fa5",
```

Comment:
I would replace this line with:
```
# fa5 is non-zero and checked in the list above.
```

Reply:
Done in https://github.com/llvm/llvm-project/pull/104547/commits/0379569846c717bcc3a0e6ecfb1c7aec4a3f218e